el-patch.el

```diff
@@ -585,6 +587,28 @@ similar."
     (list (cons 'function function-name)
           (cons 'variable variable-name))))
 
+;;;;; Font-lock functions
+
+(defun el-patch-fontify-as-defun (type)
+  "Fontify patch of TYPE as function definition."
+  (font-lock-add-keywords
+   'emacs-lisp-mode
+   `((,(concat
+        (format "(\\(el-patch-%S\\)\\>[[:blank:]]+\\(" type)
```

Comment:
Shouldn't this account for `macroname` in case it is not `nil` in `el-patch-deftype`.

Reply:
Oh yeah, that's right. I... am not sure why I implemented this `macro-name` feature in the first place, but yes, it should be supported here as long as we've got it.